### PR TITLE
Fix the formatting of the 'formatting' section

### LINKED ISSUE #871: In the formatting section, the JS example renders incorrectly
The README's formatting section currently looks like this:

image

I think the JS example ought to look like this:
````md
```js
var express = require('express')
var app = express()
app.listen(3000)
```
````

instead of this:

```js
var express = require('express')
var app = express()
app.listen(3000)
```

The README is explaining how to use JS snippets within GFM, so showing a fenced GFM block would be better.

### PR DESCRIPTION
**Pull request by shawwn**
Closes #871.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +38,13 @@ Jekyll uses the [Liquid template engine](http://liquidmarkup.org/) for templatin
...
 
+````md
 ```js
 var express = require('express')
 var app = express()
 app.listen(3000)
 ```
+````
 
 The default GitHub Pages syntax highlighting has been disabled in `_config.yml to allow highlighting with prism.js.
```
Comment (crandmck) [inline README.md:49]: As long as you're at it, could you please fix this unrelated issue --- add the terminating backtick... s/be `_config.yml` .. thx!

**Review by crandmck — CHANGES_REQUESTED** [review]
Thanks for the PR.  I like your suggestion, but I think it would be even better if you added a code block showing the result, e.g. 

The result looks like this:

```js
var express = require('express')
var app = express()
app.listen(3000)
```

**Comment by crandmck** [comment]
I made these changes with 633ae4e238514025b85537532bfea8305d54d3ce

Thanks for raising the issue @shawwn

**Comment by shawwn** [comment]
I kind of wanted to be listed as a contributor to the repo. :) Oh well.

I didn't understand your reply. Sorry about that. I see now what you were asking for.

**Comment by crandmck** [comment]
No worries, @shawwn 
I'd be happy to accept other contributions from you, if you find anything else that needs fixing.  Then you could be a contributor. :-)

**Pull request closed without merging**